typescript/sdk/src/proxy.ts

```diff
@@ -47,6 +47,10 @@ export class ProxiedContract<
     return this.contract.address;
   }
 
+  get deployTransaction() {
```

Comment:
:(

Reply:
why sad face?

Reply:
Oh just that we had to add this to ProxiedContract b/c it's not transparent.

To be clear I wouldn't do anything different